core/src/test/kotlin/fr/sncf/osrd/stdcm/StopTests.kt

```diff
@@ -91,6 +91,7 @@ class StopTests {
         )
     }
 
+    // quand le dernier step nest pas un stop on a besoin de forcer larret
```

Comment:
Accidental French

(I'm a little confused by this message)

Reply:
done